Swift/fibonacci/fibonacci.swift

```diff
@@ -1,12 +1,24 @@
-func fibonacci(n: Int) -> Int {
-var a = 0
-var b = 1
-for _ in 0..<n {
-let temp = a
-a = b
-b = temp + b
+func FibonacciSequence(n: Int) -> Int {
+    var a = 0
+    var b = 1
+    for _ in 0..<n {
+        let temp = a
+        a = b
+        b = temp + b
+    }
+    return a
```

Comment:
why not just use recursion like everywhere else?

Reply:
This was what was already there

Reply:
Done